### PR TITLE
0x7 - Add descriptions on Security Testing and clarified SDLC phases

### PR DESCRIPTION
**Pull request by romualdszkudlarek**
plus fixed some typos

### CODE REVIEW

File: Document/0x07-Security-Testing-SDLC.md [hunk 1]
```diff
@@ -67,11 +67,18 @@ Based on the risks of the project, some artefacts may be simplified (or even ski
...
+* Coding phase: this is the moment when code is produced and efforts become visible. This may be seen as the most important phase; however, one must keep in mind that all activities happening before and after the current phase are meant to support code creation and make sure it reaches proper standards for quality and security while meeting deadlines and budgets. In this phase, development teams work in the defined environnement to implement requirements following previously defined guidelines. The main people who are involved are developers.
+* Testing phase: this is the phase when produced software is tested. As testing can take many forms (see detailed section on Security Testing in the SDLC below), testing activities may be performed during coding (the obvious goal being to discover issues as soon as possible). Depending on organizations, the project risk profile and techniques used, testing teams may be independent from coding teams. The main function involved during this phase is Testers.
```
Comment (sushi2k) [inline Document/0x07-Security-Testing-SDLC.md]: Last sentence should be: "The main people involved..."

File: Document/0x07-Security-Testing-SDLC.md [hunk 2]
```diff
@@ -67,11 +67,18 @@ Based on the risks of the project, some artefacts may be simplified (or even ski
 
 #### Diving into phases and artefacts
 
--- TODO :  Quick description of all phases (what? when? who? ...) --
+Now, let's have a closer look at the five phases listed above and let's clarify their main purposes, what is done while they take place and who performs them:
+* Initiation phase: this is the first phase of a project, when requirements are gathered from the field and defined for the project. They should include both functional (e.g. what fonctions will be created for the end user) and security (e.g. what security features will need to be implemented to allow end users to trust the software product) requirements. In this phase, all activities that need to happen before technical work starts and all others that can be anticipated will take place. This is also the moment when Proof of Concepts may be done and when the project viability is confirmed. Typically, teams close to business functions such as Marketing (marketing people, or Product Owners, ...), Management and Finance are involved.
```
Comment (sushi2k) [inline Document/0x07-Security-Testing-SDLC.md]: typo: fonctions should be functions

File: Document/0x07-Security-Testing-SDLC.md [hunk 3]
```diff
@@ -67,11 +67,18 @@ Based on the risks of the project, some artefacts may be simplified (or even ski
...
+
+Even if the previous description seems to be "Waterfall-like", it also applies to Agile methodologies: the same logic is used, but in a more iterative manner. Some activities may be done only once (for instance project initiation), however smaller parts of similar activities will happen regularly all along the project (like bringing new requirements into light and clarifying them into Stories). In the same manner, testing will ont happen only once at the end of a project, but, on each iteration, tests will focus on the amount of code that was produced in the iteration.
```
Comment (sushi2k) [inline Document/0x07-Security-Testing-SDLC.md]: typo: ont should be not

**Comment by romualdszkudlarek** [comment]
Thanks for your feedback, will implement these changes.

**Comment by romualdszkudlarek** [comment]
Done with changes

**Comment by b-mueller** [comment]
Great work, thanks!

**Pull request merged**